core/store/src/metrics.rs

```diff
@@ -205,3 +205,12 @@ pub static PREFETCH_STAGED_SLOTS: Lazy<IntGaugeVec> = Lazy::new(|| {
     )
     .unwrap()
 });
+#[cfg(feature = "cold_store")]
+pub static COLD_MIGRATION_READS: Lazy<IntCounterVec> = Lazy::new(|| {
+    try_create_int_counter_vec(
+        "near_cold_migration_reads",
+        "Number of get calls to hot store made for every column during copying data to cold storage.",
+        &["column"],
```

Comment:
```suggestion
        &["col"],
```
We call the label `col` in RocksDB metrics so let’s keep things consistent.